packages/wrangler/src/cloudchamber/common.ts

```diff
@@ -119,11 +119,11 @@ export function handleFailure<
 			}
 
 			if (err instanceof Error) {
-				logger.log(`${JSON.stringify({ error: err.message })}`);
+				logger.log(JSON.stringify({ error: err.message }));
 				return;
 			}
 
-			logger.log(JSON.stringify(err));
+			throw err;
```

Comment:
I note that if the `err` is a `UserError` then it will just be JSON logged and the process will exit with a 0 exit code. Is this what you want?